### ISSUE TITLE
Missing Years on Product Page

### ISSUE THREAD

**Issue by jennyem56** [CARLI/select#97]
When on a product page (for example https://select-staff.carli.illinois.edu/product/3414a3cc-a714-4fcf-b765-f0e9c2060f9e/0375e79a-d53e-43bf-9e57-038c34a112e5), some years are missing under subscriber information.

For fiscal year products, it is missing 2013, 2016, and 2017.  For calendar year products, it is missing 2011, 2015, 2016, and 2018.  I've looked at different cycles and different products and they are all missing those years.
![missing years](https://user-images.githubusercontent.com/46798768/53763091-770b8580-3e8f-11e9-86c4-bddecab24e55.png)

**Comment by nicolemoorepixo** [comment]
Hi Jenny, 

Can you clarify what you would expect to see here based on the data? Is there 2013 information for a fiscal year product you're looking for but can't get to because it's not listed?

Thanks!

**Comment by jennyem56** [comment]
Yes, it should be a list of the past 5 years and their number of subscribers.  It used to be that way, but in the past week we've noticed a lot of missing data.

**Comment by CClennon** [comment]
I rechecked this today on the staging server.  What am I seeing is still wrong, but is more consistent.  I checked past subscription data for both Aurora University and Northwestern University.  I looked at the following products:

Chronicle of Higher Education
Essay and General Literature Index
Global Issues in Context
HNP Newsday

Here are the years that I saw for both libraries and all products:

2019
2018
2017
2016
2011

2015 should be the earliest year shown here.

**Comment by nicolemoorepixo** [comment]
Transferred to JIRA as CS-15
